sdk/metricsadvisor/ai-metrics-advisor/test/public/hookTests.spec.ts

```diff
@@ -61,7 +61,7 @@ matrix([[true, false]] as const, async (useAad) => {
           name: webHookName,
           description: "description",
           hookParameter: {
-            endpoint: "https://mawebhook.azurewebsites.net/api/HttpTrigger",
+            endpoint: "https://httpbin.org/post",
```

Comment:
I remember @anuchandy had to create the function app for this. Glad to see it's not needed any more?

Reply:
ooh - not sure about that but I saw Java's hook tests and they use these endpoints. Those function app endpoints were not working anymore.

Reply:
Anu said that someone stopped the host.